components/LogoNav/LogoNav.tsx

```diff
@@ -11,15 +12,17 @@ import logo from '/public/assets/logo-colored-nav.svg';
  */
 export const LogoNav = (): JSX.Element => {
   return (
-    <Image
-      src={logo}
-      alt="Gridiron Survivor logo"
-      width={1}
-      height={1}
-      priority
-      className="w-20"
-      data-testid="logo-nav"
-    />
+    <Link href="/league/all" passHref>
```

Comment:
What does `passHref` do?

Reply:
Per this Stackoverflow blog I was reading, it stated `passHref` is needed for a functional component per the nextjs documentation, but as I'm reading this again, it stated to 'pass it whenever you use a react component as a child' and for this logo instance, there are no children now that I think about it and will be removing the `passHref`. 

https://stackoverflow.com/questions/68983089/is-it-necessary-to-pass-passhref-to-nextjs-link-having-a-semantic-ui-react-a